tool/generate-specs.sh

```diff
@@ -50,6 +50,12 @@ function generate_spec() {
   cd external/nextcloud-drop_account
   generate_spec "." "drop_account"
 )
+(
+  yq -s '.[0]."components"."schemas" = .[1] | .[0] | walk(if type == "object" then with_entries( if (.key == "$ref" and (.value | test("#\/components\/schemas\/") | not)  ) then .value |= sub("#\/"; "#/components/schemas/") else . end ) else . end) | .paths |= with_entries(.key = "/index.php/apps/cookbook\(.key)") ' \
+    external/cookbook/docs/dev/api/0.1.2/openapi-cookbook.yaml \
+    external/cookbook/docs/dev/api/0.1.2/objects.yaml \
+    > /tmp/nextcloud-neon/cookbook.openapi.json
+)
```

Comment:
Until now we said every new spec can only be added if it was generated by openapi-extractor. For notes and news we still have handwritten once, but they are legacy and work fine atm. If we add this we might need to rethink what our stance on this is. For me personally this is fine, since the spec seems well maintained and we can reason that the quality is good and any fixes will be accepted upstream.

Reply:
As long as the specs are high enough quality and cover 100% of the api we should allow them.
AFIK Christian wants to move to the openapi-extractor eventually but as always that'll take some time :)